pep-0699.rst

```diff
@@ -2,7 +2,7 @@ PEP: 699
 Title: Remove private dict version field added in PEP 509
 Author: Ken Jin <kenjin@python.org>
 Discussions-To: https://discuss.python.org/t/pep-699-remove-private-dict-version-field-added-in-pep-509/19724
-Status: Draft
+Status: Final
```

Comment:
```suggestion
Status: Accepted
```

The status should be Accepted rather than Final, until the PEP is fully implemented (i.e. the version field is fully deprecated and replaced in a released Python version